docs/conf.py

```diff
@@ -117,6 +117,15 @@
 .. |icon-color-square| image:: /img/icons/color_square.png
   :scale: 40
   :alt: color square icon
+
+.. |icon-box-zoom| image:: /img/icons/box_zoom.png
```

Comment:
we really should link directly to the icon images at some point now that almost all are custom... but the rest are screenshots, so let's consider that out-of-scope for now and try to update them all at once at some point in the future?

Reply:
Sounds good. I wasn't sure if SVG files would work here so I defaulted to screenshotting for now. Definitely something to explore further to reduce duplicate files.